recipes/mapd-core-cpu/build.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# sed -i option is handled differently in Linux and OSX
```

Comment:
I prefer adding `sed` to build requirements, which gives you GNU sed so that you don't have to deal with GNU vs BSD sed

Reply:
sounds good! we will change that in the feedstock!